src/pages/[language]/blog/[slug]/index.vue

```diff
@@ -279,6 +282,26 @@
     margin-bottom: var(--spacing-smaller);
   }
 
+  .page-blog-post-list__title--h2 {
```

Comment:
we have these styles already defined in typography, why are we doing typography styling again?